### PR TITLE
Fix typo in README.md

### PR DESCRIPTION
**Pull request by Chandanmali**
### Summary
change wrong words
change grammer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ some packages that provide implementations of a runtime primitive implementation
 
 * https://github.com/Qiskit/qiskit-ibm-runtime
 
-Qiskit also provides a lower-level abstract interface for describing quantum backends. This interface, located in
+Qiskit also provides a lower-level abstract interface for describing quantum backends. This interface, is located in
```
Comment (mtreinish) [inline README.md]: This is actually correct usage with the commas surrounding it.

File: README.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ pip install qiskit
 
 Pip will handle all dependencies automatically and you will always install the latest (and well-tested) version.
 
-To install from source, follow the instructions in the [documentation](https://qiskit.org/documentation/contributing_to_qiskit.html#install-install-from-source-label).
+To install from the source, follow the instructions in the [documentation](https://qiskit.org/documentation/contributing_to_qiskit.html#install-install-from-source-label).
```
Comment (mtreinish) [inline README.md]: "install from source" is a pretty common phrasing to describe installing a program from its source code.

File: README.md [hunk 3]
```diff
@@ -157,13 +157,13 @@ release.
...
 document in detail what has changed as part of a release. This includes any
-documentation on potential breaking changes on upgrade and new features.
+documentation on potential breaking changes on upgrades and new features.
 For example, you can find the release notes for the `0.9.0` release in the
 Qiskit documentation here:
 
 https://qiskit.org/documentation/release_notes.html#terra-0-9
 
-## Acknowledgements
+## Acknowledgments
```
Comment (mtreinish) [inline README.md]: Both of these are valid spellings of the word, I'm not sure we need to change it.

File: README.md [hunk 4]
```diff
@@ -124,7 +124,7 @@ hardware or simulators to Qiskit. The backend class includes a common interface
 <!-- This is not an exhaustive list, and if you maintain a provider package please feel free to open a PR to add new providers -->
 
 You can refer to the documentation of these packages for further instructions
-on how to get access and use these systems.
+on how to get access to use these systems.
```
Comment (mtreinish) [inline README.md]: I feel like the previous wording is more descriptive as it's distinguishing that getting access and using the systems are different tasks.

File: README.md [hunk 5]
```diff
@@ -157,13 +157,13 @@ release.
 
 Additionally, as part of each release detailed release notes are written to
 document in detail what has changed as part of a release. This includes any
-documentation on potential breaking changes on upgrade and new features.
+documentation on potential breaking changes on upgrades and new features.
```
Comment (mtreinish) [inline README.md]: The singular is correct here, because "upgrade" is the action being taken.

**Review by mtreinish — CHANGES_REQUESTED** [review]
Thanks for taking the time to draft a pull request and submit it and your first contribution to Qiskit! I left some comments inline, but I think most of the typos you're correcting aren't actually typos or issues in the README text. There was the one change to capitalization on L74 that I think was fine though. So if you reduce the changes in your PR so it's just that single line this will be ok to merge.

**Review by 1ucian0 — APPROVED** [review]
I left the L74 change only.

**Review by mtreinish — APPROVED** [review]

**Pull request merged**